plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -208,6 +209,17 @@ func ClusterRoles() []rbacv1.ClusterRole {
 				).RuleOrDie(),
 			},
 		},
+		{
+			// a role which provides minimal read access to the instrumentation endpoints
+			// (i.e. /metrics, /debug/pprof/*")
+			ObjectMeta: metav1.ObjectMeta{Name: "system:monitoring"},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule("get").URLs(
+					"/metrics",
+					"/debug/pprof/*",
```

Comment:
hrm, I'm a little surprised to see /debug here. That seems more powerful than /metrics. Also is /debug useful if you can only send GET requests?

What about adding /healthz*, /livez*, /readyz*, componentstatuses (ugh), /version, and /logs?

Reply:
ahahaha you mentioned, by name, the endpoint that must remain unmentioned by name. 

Yeah I can add the other ones but for the other one I will defer to @liggitt (whom I tried unsuccessfully to convince of this earlier)

Reply:
yeah, I really don't like a built-in role for this... I think people should define a role for their monitoring component that includes just the endpoints they want

Reply:
Had out-of-band conversation with @liggitt, we've agreed to leave out pprof and the endpoint that must not be referred to by name but added the healthcheck splats for the error reporting.

(Also, we actually don't need version since it's already accessible by public-info-viewer to unauthenticated.)